test/e2e/Tests/e2e-tests-readme.md

```diff
@@ -0,0 +1,59 @@
+# End-to-End Test Project
+
+This document provides instructions on how to use the end-to-end (E2E) test project for the Azure Functions Durable Extension.
+
+## Prerequisites
+
+- Visual Studio IDE
```

Comment:
Is Visual Studio required or just recommended? Based on what I'm seeing, it doesn't look like we strictly require an IDE to run tests.

Reply:
Removed this line